spec/features/create_extension_spec.rb

```diff
@@ -94,6 +95,25 @@ def check_create_extension
     end
   end
 
+  def check_factories_moved_to_new_location
+    lib = Pathname(File.join(install_path, "lib/solidus_test_extension"))
+
+    # Simulate what we can find on old extensions
+    FileUtils.mv(lib.join("testing_support/factories.rb"), lib.join("factories.rb"))
+    FileUtils.rmdir(lib.join("testing_support"))
+    FileUtils.mkdir(lib.join("factories"))
+
+    cd(install_path) do
+      # Executing the `solidus extension .` command again, will ask user prompt
+      # to override the factories file we moved in the new postion.
+      expect(Thor::LineEditor).to receive(:readline).and_return("n")
+      sh('solidus extension .')
+
+      expect(lib.join("testing_support/factories.rb")).to exist
+      expect(lib.join("testing_support/factories")).to exist
```

Comment:
@elia this expectation is failing, I'm not sure how to debug this properly. When you have some time, can you please help me?